cmd/ttn-lw-cli/commands/end_devices.go

```diff
@@ -536,8 +537,12 @@ var (
 			if err != nil {
 				return err
 			}
+			extJS, err := cmd.Flags().GetBool("external-join-server")
+			if err != nil {
```

Comment:
Afaik we don't need errors check here, just `extJS, _ := ...`

Reply:
We're not always consistent with this. AFAIK the only way this can error is if the flag isn't defined, or if it's defined as a string (or any other type) and try to read it as a bool

Reply:
In either case, we don't have to check errors here. If people pass a `--external-join-server="the world is flat"`, they won't enable the external JS.

Reply:
IMO the user should know that the flag value is dropped instead of silently ignored. E.g. if user passes `--external-join-server=tru` (typo) and that is silently ignored, how does user know that his command actually failed?

Reply:
OK fine